the-codeless-code/fr-meowcate/case-230.txt

```diff
@@ -0,0 +1,84 @@
+Title: Tous ensemble désormais
+Tagline: Où l'on sous-traite certaines choses
+Lang: fr
+Translator: Meowcate
+
+Le [[Hungry Worm Clan|Clan du Ver Affamé]] construisait un site web
+pour un artisan qui fabriquait des outils agricoles spécialisés.
+La jeune maître [[Zjing]] était en pleine revue de code de ses trois
+développeurs.
+
+"Je ne comprends pas l'intérêt de la propriété /DateLimiteDeVente/ dans
+/AcheterProduit/ de votre panier," dit-elle aux trois.
+"Les pelles et les râteaux n'expirent pas."
+
+"C'est une propriété qui nous a été demandée par le
+[[Red Pebble Clan|Clan du Caillou Rouge]]," répondit le premier moine.
```

Comment:
I can agree on that part. Although you fixed some spelling mistakes, you added others, as well as Anglicisms. Also, those anglicisms -to me- somewhat change the usually solemn "tone", that's why I usually avoid them. I'll correct my mistakes and use your proposed name for the `Pebble Clan`

Reply:
I'm ok with that. Also, the automatic rule as I read it is "first in is published by default", so corrections on yours is better. I did my version because yours was old, I wasn't sure you would still be on it.

Reply:
That's really nice of you to participate! I've been indeed inactive but still watching the repository. I didn't even know this rule but if we stick with it, please comment on mine if you see any mistake I'd have left there.
As for the english terms, I do prefer sticking with "non-industry" words because of the general tone. I find it quite disruptive to change from "soutenu" to anglicism. It could be nice to have some more global decision about that though, maybe from @QilessQi or other french translators like @cdlm

Reply:
In fact of a "rule", it's apparently how the updating translation system works everytimes it's launched :)
For the rest, I'm ok with any change, my most important concern is it feels consistant with the others translations.